Python/Product/PythonTools/PythonTools/WebProject/PythonWebLauncherProvider.cs

```diff
@@ -104,7 +104,7 @@ string environmentProperty
                 if (string.IsNullOrEmpty(config.ScriptArguments)) {
                     config.ScriptArguments = args;
                 } else {
-                    config.ScriptArguments = config.ScriptArguments + " " + args;
+                    config.ScriptArguments = args + " " + config.ScriptArguments;
```

Comment:
This was needed to prevent django launch from breaking if script arguments were provided:
![image](https://user-images.githubusercontent.com/3840081/39277338-247cdcc6-48a2-11e8-9723-925d7e0b5d64.png)

This does not fix the issue where we may have script arguments repeated twice. However, the user can remove `$(CommandLineArgs)` from the end of `DebugWebServerArguments` or `RunWebServerArguments` to fix that.